routers/api/v1/repo/collaborators.go

```diff
@@ -238,6 +245,9 @@ func DeleteCollaborator(ctx *context.APIContext) {
 		ctx.Error(http.StatusInternalServerError, "DeleteCollaboration", err)
 		return
 	}
+
+	audit.Record(ctx, audit_model.RepositoryCollaboratorRemove, ctx.Doer, ctx.Repo.Repository, collaborator, "Removed collaborator %s.", collaborator.Name)
```

Comment:
I'm unsure how much info we want in the string because we have scope and target to get/display more information.
![grafik](https://github.com/go-gitea/gitea/assets/1666336/b130ed7f-2a68-4f32-bc4d-8f06adfced41)

Reply:
Okay, I haven't seen it in the web view yet.
Nevertheless, users using the JSON file don't have this information.

Reply:
Also, as you've shown in your own screenshot, the information is lost as soon as the team, org, user, or repo is deleted.